packages/jest-circus/src/utils.ts

```diff
@@ -189,9 +190,23 @@ export const callAsyncCircusFn = (
     // soon as `done` called.
     if (takesDoneCallback(fn)) {
       let returnedValue: unknown = undefined;
+
       const done = (reason?: Error | string): void => {
+        if (!completed && testOrHook.seenDone) {
+          let message =
+            'Expected done to be called once, but it was called multiple times.';
+
+          if (reason) {
+            message += ' Reason: ' + reason.toString();
```

Comment:
```suggestion
            message += ' Reason: ' + prettyFormat(reason, {maxDepth: 3});
```

to match what we do further down